apps/theme/package.json

```diff
@@ -16,7 +16,6 @@
     "@digdir/designsystemet-theme": "workspace:^",
     "@navikt/aksel-icons": "^6.14.0",
     "@repo/components": "workspace:^",
-    "chroma-js": "^2.6.0",
```

Comment:
This is used, but in a component that is not used.
Can you delete `FullBaseTest.tsx`?

Reply:
I assume as much? https://pr-3016.theme.designsystemet.no/testside

Its used in `generateColorScale`, which is also used in `tokens create` cli cmd, and what was used to update the design-tokens colors. Looked like only some tweaks to the hex codes.

Reply:
Fixed!